supertokens_python/recipe/dashboard/interfaces.py

```diff
@@ -98,3 +98,132 @@ def to_json(self) -> Dict[str, Any]:
             "users": users_json,
             "nextPaginationToken": self.next_pagination_token,
         }
+
+
+class DashboardUsersGetResponseWithMetadata(APIResponse):
+    status: str = "OK"
+
+    def __init__(
+        self, users: List[Dict[str, Any]], next_pagination_token: Optional[str]
+    ):
+        self.users = users
+        self.next_pagination_token = next_pagination_token
+
+    def to_json(self) -> Dict[str, Any]:
+        return {
+            "status": self.status,
+            "users": self.users,
+            "nextPaginationToken": self.next_pagination_token,
+        }
+
+
+class UserCountGetAPIResponse(APIResponse):
+    status: str = "OK"
+
+    def __init__(self, count: int):
+        self.count = count
+
+    def to_json(self) -> Dict[str, Any]:
+        return {"status": self.status, "count": self.count}
+
+
+class UserGetAPIOkResponse(APIResponse):
+    status: str = "OK"
+
+    def __init__(self, recipe_id: str, user: Dict[str, Any]):
```

Comment:
The user type should be strongly typed, not just a key value pair

Reply:
Fixed.